pages/index.tsx

```diff
@@ -14,7 +19,23 @@ const Home: NextPage = () => {
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Layout>
+      <Layout hideHeaderSearch={true} marginY={"130px"}>
+        <Stack align={"center"}>
+          <HStack spacing={5} align={"center"}>
+            <Image
+              src={bookBazarLogo}
+              alt="Book Bazar Logo"
```

Comment:
Alt text should be `alt=""` as this is just for display (https://www.w3.org/WAI/tutorials/images/decorative/)